### PR TITLE
Update all browsers versions for ImageData API

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects the real values for all browsers for the `ImageData` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v4.0.0).

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/ImageData

_Check out the [collector's guide on how to review this PR](https://github.com/foolip/mdn-bcd-collector#reviewing-bcd-changes)._

### CODE REVIEW

File: api/ImageData.json [hunk 1]
```diff
@@ -15,10 +15,10 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": "14"
+            "version_added": "3.5"
```
Comment (foolip) [inline api/ImageData.json:18]: This is still later than the api.CanvasRenderingContext2D.createImageData entry. They should match, so can you check which is correct?

File: api/ImageData.json [hunk 2]
```diff
@@ -55,10 +55,10 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#dom-imagedata-dev",
           "support": {
             "chrome": {
-              "version_added": "42"
+              "version_added": "36"
```
Comment (foolip) [inline api/ImageData.json:58]: Should this be mirrored to Opera too?

File: api/ImageData.json [hunk 3]
```diff
@@ -161,10 +161,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "14"
+              "version_added": "3.5"
```
Comment (foolip) [inline api/ImageData.json:164]: This and the following features should probably match the parent feature if that changes to match `createImageData`.

**Review by foolip — CHANGES_REQUESTED** [review]

File: api/ImageData.json (hunk 1 above)
Comment (queengooborg) [reply api/ImageData.json:18]: Our test uses that function to create an `ImageData` instance, and it appears it isn't defined in Firefox 3.  Will correct that data in this PR!

**Review by foolip — APPROVED** [review]

**Pull request merged**